www/simple.php

```diff
@@ -253,9 +253,13 @@ function getScheme() {
 	$hubic_user = $_GET['hubic_user'];
 	$hubic_password = $_GET['hubic_password'];
 
+        if (is_array($clients[$client]) && array_key_exists('autoredirect_uri', $clients[$client])) {
```

Comment:
`isset()` suppresses the warning. You could also use something like `@array_key_exists('autoredirect_uri', $clients[$client])`  which also will suppress the warning. Which I do not really like. In addition `isset()` is normally used to check if a variable exists. I also do not like that :) .. It's usage for me is basically just this kind of checking arrays.